x-pack/platform/plugins/shared/ml/public/application/model_management/test_models/models/ner/ner_output.tsx

```diff
@@ -11,53 +11,62 @@ import React from 'react';
 import useObservable from 'react-use/lib/useObservable';
 import { FormattedMessage } from '@kbn/i18n-react';
 import {
+  useEuiFontSize,
+  useEuiTheme,
   EuiBadge,
   EuiFlexGroup,
   EuiFlexItem,
   EuiHorizontalRule,
   EuiIcon,
   EuiToolTip,
+  type EuiThemeComputed,
 } from '@elastic/eui';
-import { useCurrentThemeVars } from '../../../../contexts/kibana';
-import type { EuiThemeType } from '../../../../components/color_range_legend/use_color_range';
 import type { NerInference, NerResponse } from './ner_inference';
 import { INPUT_TYPE } from '../inference_base';
 
+type EuiVisColor = keyof EuiThemeComputed['colors']['vis'];
+
 const ICON_PADDING = '2px';
 const PROBABILITY_SIG_FIGS = 3;
 
-const ENTITY_TYPES = {
+interface EntityType {
+  label: string;
+  icon: string;
+  color: EuiVisColor;
+}
+
+const ENTITY_TYPE_NAMES = ['PER', 'LOC', 'ORG', 'MISC'] as const;
+const isEntityTypeName = (name: string): name is EntityTypeName =>
+  ENTITY_TYPE_NAMES.includes(name as EntityTypeName);
+type EntityTypeName = (typeof ENTITY_TYPE_NAMES)[number];
+
+const ENTITY_TYPES: Record<EntityTypeName, EntityType> = {
   PER: {
     label: 'Person',
     icon: 'user',
-    color: 'euiColorVis5_behindText',
-    borderColor: 'euiColorVis5',
+    color: 'euiColorVis5',
   },
   LOC: {
     label: 'Location',
     icon: 'visMapCoordinate',
-    color: 'euiColorVis1_behindText',
-    borderColor: 'euiColorVis1',
+    color: 'euiColorVis1',
```

Comment:
Hi @walterra, visColors mapping has changed in Borealis (as seen [here](https://docs.google.com/document/d/1IAKbasq1nDfqd2IU3KdP8cwD3uCCAwkIekKRq7zgyWg/edit?usp=sharing)):

<img width="617" alt="image" src="https://github.com/user-attachments/assets/2f7bb7d9-12f9-4abe-aa68-df275e8388d9" />

So here if you want to preserve the blue you'll probably want to do  `euiColorVis2` instead.

Reply:
@walterra @andreadelrio Fyi, changing colors like this means the current color in Amsterdam changes as well. (in Amsterdam it will be red now)
If that's not wanted, we'd need to use conditional color definitions.